### PR TITLE
Add recipe for tornado-sqlalchemy

### PR DESCRIPTION
**Pull request by lincrosenbach**

### CODE REVIEW

File: recipes/tornado-sqlalchemy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  license_file: 
+  summary: SQLAlchemy helpers for working in Tornado
+  doc_url: https://tornado-sqlalchemy.readthedocs.io/
+  dev_url: https://github.com/siddhantgoel/tornado-sqlalchemy
+
+extra:
+  recipe-maintainers:
+    - lincrosenbach
+    - kfasolin
+    - nicoddemus
```
Comment (beckermr) [inline recipes/tornado-sqlalchemy/meta.yaml]: @kfasolin  and @nicoddemus need to confirm they are ok maintaining the recipe.

Comment (nicoddemus) [reply recipes/tornado-sqlalchemy/meta.yaml]: Sure 👍

File: recipes/tornado-sqlalchemy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+{% set name = "tornado-sqlalchemy" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+#https://files.pythonhosted.org/packages/d9/4b/02fc7c444590a077aa4f0a5fbf8380be07d71a30fd435751fbfa4fa6a775/tornado_sqlalchemy-0.5.0.tar.gz
```
Comment (nicoddemus) [inline recipes/tornado-sqlalchemy/meta.yaml]: Please remove this comment

Comment (lincrosenbach) [reply recipes/tornado-sqlalchemy/meta.yaml]: Done.

**Pull request merged**